data/default_os_images.json

```diff
@@ -80,7 +80,7 @@
     {
         "openshift_version": "4.13",
         "cpu_architecture": "arm64",
-        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso",
+        "url": "`https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso`",
```

Comment:
why this change is related to this PR?

Reply:
Not related, probably key was pressed by mistake

Reply:
Fixed